docs/modules/plugins/pages/plugin-azure-resource-manager.adoc

```diff
@@ -30,6 +30,45 @@ The supported environments are only:
 Azure subscription must be configured via `AZURE_SUBSCRIPTION_ID` environment variable.
 
 
+=== Secrets Management Tools
+
+==== https://learn.microsoft.com/en-us/azure/key-vault/[Azure Key Vault]
+
+===== Configuration
+
+NOTE: The properties marked with *bold* are mandatory.
+
+[cols="1,3,1", options="header"]
+|===
+|Property
+|Description
+|Example
+|Default value
+
+|[subs=+quotes]`*azure.key-vault.name*`
+|Azure Key Vault Name with secret values for properties
```

Comment:
it's needed to describe how to get this name

Reply:
added particular section with instruction